packages/viewtoggle/src/react/__specs__/index.spec.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { render } from '@testing-library/react'
 
-import ViewToggle from '../index.js'
```

Comment:
hehe, remember when we went through all the work to do the new/non-magic module resolution way?